src/main/config/config_eeprom.h

```diff
@@ -20,7 +20,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 
-#define EEPROM_CONF_VERSION 162
+#define EEPROM_CONF_VERSION 163
```

Comment:
Thanks!

Reply:
I saw the version change but was really questioning the logic behind having the dterm_setpoint_weight defaulting to zero.  If it was intentional, then we'll need to be sure to call it out in the change notes for RC5 (or whatever comes next) as it will affect a lot of people.  Similarly to when the default changed from 100 to 60 not that long ago and caught a bunch of people out.